src/components/idleUsers/card/card.module.scss

```diff
@@ -8,16 +11,23 @@
   flex-wrap: wrap;
   margin: 2rem;
   padding: 2rem;
-  cursor: pointer;
```

Comment:
was this intentional?
the cursor pointer gives the feel of a clickable object.

Reply:
Yes